epinar/assignment4/islands.py

```diff
@@ -0,0 +1,56 @@
+import unittest
+
+
+def visit_island(grid, k, l, M, N):
+	""" Marks the visited island pieces to not to visit in the future again."""
+	if 0 <= k < M and 0 <= l < N:
+		if grid[k][l] is True:
```

Comment:
nit: simply 

if grid[k][l]:

"if grid[k][l]" and "if grid[k][l] is True" are equivalent and the latter is more verbose.